src/dev/precommit_hook/casing_check_config.js

```diff
@@ -26,6 +26,7 @@ export const IGNORE_FILE_GLOBS = [
   '**/{Dockerfile,docker-compose.yml}',
   'x-pack/plugins/canvas/tasks/**/*',
   'x-pack/plugins/canvas/canvas_plugin_src/**/*',
+  'x-pack/plugins/cases/docs/**/*',
```

Comment:
Ignoring the cases plugin's docs directory because the files are generated and won't necessarily be in snake case.